services/rds/rds.go

```diff
@@ -502,6 +502,12 @@ func (svc *Service) updateRDSExporterConfig(tx *reform.TX, service *models.RDSSe
 	}
 	for _, n := range nodes {
 		node := n.(*models.RDSNode)
+
+		service := &models.RDSService{}
+		if e := tx.SelectOneTo(service, "WHERE node_id = ?", node.ID); e != nil {
```

Comment:
```go
tx.FindOneTo(&service, "node_id", node.ID)
```
We can and should avoid SQL strings with MySQL-specific placeholders in simplest cases like that one.